tools/generators/move-packages/index.ts

```diff
@@ -154,3 +163,18 @@ function updateCodeOwners(tree: Tree, schema: MovePackagesGeneratorSchema) {
     tree.write(codeownersPath, updatedCodeOwnersFile);
   }
 }
+
+function updateApiExtractor(tree: Tree, schema: MovePackagesGeneratorSchema) {
+  const projectConfig = getProjectConfig(tree, { packageName: schema.name! });
+  const apiExtractorLocalPath = joinPathFragments(projectConfig.paths.configRoot, 'api-extractor.local.json');
+  const apiExtractorLocal = readJson(tree, apiExtractorLocalPath);
+  const originalEntryPointPath = apiExtractorLocal.mainEntryPointFilePath;
+  const normalizeFolderDestination = schema.destination.split('/').slice(0, -1);
```

Comment:
This is currently returning an array instead of a string
```suggestion
  const normalizeFolderDestination = schema.destination.split('/').slice(0, -1).join("/");
```